contrib/ansible/roles/master/tasks/main.yml

```diff
@@ -1,18 +1,42 @@
 ---
+- name: Assures /srv/kubernetes/ dir exists
+  file: path=/srv/kubernetes/ state=directory
+
+- name: Create system kube-cert groups
+  group: name=kube-cert state=present system=yes
```

Comment:
I believe the kube-apiserver needed this group applied to read the files. I am not sure on if that has changed. I will look into this.